shared/teams/team/invites/index.js

```diff
@@ -30,8 +30,23 @@ const DividerRow = (index, {key}) => (
   </Box>
 )
 
-const TeamRequestOrDividerOrInviteRow = (index, row) => {
-  switch (row.type) {
+export const renderRequestsOrInvitesRow = (index: number, row: RequestsOrInvitesRow) => {
```

Comment:
They also export a row renderer that handles all cases, which are called in the main row renderer in `list.js`